src/sections/SideBar/components/NavLinks/SideBarLinks.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import HomeOutlinedIcon from "@material-ui/icons/HomeOutlined";
+import PersonOutlineOutlinedIcon from "@material-ui/icons/PersonOutlineOutlined";
+import MailOutlineOutlinedIcon from "@material-ui/icons/MailOutlineOutlined";
+import NotesOutlinedIcon from "@material-ui/icons/NotesOutlined";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core/styles";
+import Link from "@material-ui/core/Link";
```

Comment:
Verify it's possible to do the same here with material ui, so that we can reduce the amount of imports

Reply:
Something like this?:

```suggestion
import { makeStyles, List, ListItem, ListItemIcon, ... } from "@material-ui/core";
import { HomeOutlinedIcon, PersonOutlineOutlinedIcon, ... } from "@material-ui/icons";
```

Reply:
exactly